grapple/helpers.py

```diff
@@ -409,7 +413,7 @@ def resolve_plural(self, _, info, **kwargs):
                 if paginated:
                     return resolve_paginated_queryset(qs.all(), info, **kwargs)
                 else:
-                    return resolve_queryset(qs.all(), info, **kwargs)
+                    return resolve_queryset(qs.all(), info, limit, **kwargs)
```

Comment:
I think `limit` might already be a kwarg that can be supplied as part of the query, for example:

```graphql
articles(limit: 10) {
  slug
}
```

so this forced limit might be overridden by that. Maybe do something like:

```python
limit = max(kwargs.pop("limit", limit), limit)
```